client/src/partials/reports_proposed/debitor_group_report/debitor_group_report.js

```diff
@@ -0,0 +1,74 @@
+var DebtorGroupReportController = function ($http, $translate, validate, reportConfigService, messenger) {
+
+  var vm = this,
+  generatedDocumentPath = null,
+  serverUtilityPath = '/report/build/debitor_group_report',
+  configuration = reportConfigService.configuration,
+  dependencies = {};
+
+  dependencies.debitorGroup = {
+      query : {
+        identifier : 'uuid',
+        tables : {
+          'debitor_group' : {
+            columns : ['uuid', 'name', 'account_id', 'is_convention']
+          }
+        },
+        where : ['debitor_group.is_convention=1']
+      }
+  };
+
+  vm.generate_doc = $translate.instant('DEBITOR_GROUP_REPORT.GENERATE_DOC');
```

Comment:
Instead of using $translate.instant() here (which will not update if the translation changes), why not put this in the view?  It would go on [this line](https://github.com/IMA-WorldHealth/bhima/pull/771/files#diff-27d9fb20a3fcd83f5250cfbf4ae2e384R59), with the same logic.